src/index.js

```diff
@@ -491,36 +491,33 @@ if (nativeBindings.nativeOculusVR) {
         nativeBindings.nativeWindow.setCurrentWindowContext(windowHandle);
 
         // fps = VR_FPS;
+
         const system = vrPresentState.oculusSystem || nativeBindings.nativeOculusVR.Oculus_Init();
         const lmContext = vrPresentState.lmContext || (nativeBindings.nativeLm && new nativeBindings.nativeLm());
 
         const {width: halfWidth, height} = system.GetRecommendedRenderTargetSize();
-        const MAX_TEXTURE_SIZE = 4096;
+        /* const MAX_TEXTURE_SIZE = 4096;
```

Comment:
Should we remove?

Reply:
Yeah, no good reason to keep it. We should be 100% accurate with our resizes now.